_includes/footer.html

```diff
@@ -16,7 +16,7 @@
 					{% endfor %}
 					<ul class="feed">
 						<div class="col l4">
-							<a class="twitter-timeline" data-height="310" data-width="310" style="border-radius: 6px;"
+							<a class="twitter-timeline" data-width="320" data-height="320"
```

Comment:
The border radius should not be removed.

What's the behavioral difference that we're seeking to effect by reducing the size of the embed?

Reply:
actually :sweat_smile: they should be removed as they are overwritting already made css which is leading to this , the current design 

![image](https://user-images.githubusercontent.com/95926324/190468895-607180ad-5213-4022-86e9-1cdcb749a65d.png)